PWGLF/Tasks/Resonances/lambda1520analysis.cxx

```diff
@@ -404,7 +404,7 @@ struct lambda1520analysis {
         }
       }
     }
-    return false;
+    return isTrk1Selected;
```

Comment:
@hkoley this looks like a fixed bug, isn't it?

Reply:
Yes, it should return boolean values for `isTrk1Selected` and `isTrk2Selected`. Thanks for fixing the bug.